mpp/shmemx_c_func.h4

```diff
@@ -38,4 +38,5 @@ SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmemx_pcntr_get_all(shmem_ctx_t ctx, shme
 /* Separate initializers */
 SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmemx_heap_create(void *base, size_t size, int device_type, int device_index);
 SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmemx_heap_preinit();
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_heap_preinit_thread(int requested, int *provided);
 SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmemx_heap_postinit();
```

Comment:
@wrrobin - do we want weak symbols for these (for pshmem)?

Reply:
Oh.. yes. Thanks. I will add..